pkgs/servers/invidious/default.nix

```diff
@@ -23,7 +34,8 @@ crystal.buildCrystalPackage rec {
     owner = "iv-org";
     repo = pname;
     fetchSubmodules = true;
-    inherit (versions.invidious) rev hash;
+    rev = if versions.invidious ? rev then versions.invidious.rev else "v${version}";
```

Comment:
```suggestion
    rev = if versions.invidious.rev or "v${version}";
```
Couldn't this work ?

Reply:
oh yeah that'd work too, thanks! feel free to apply that patch yourself as it's on your branch anyway :)

Reply:
Applied the patch !